content/docs/1.6.0/volumes-and-nodes/maintenance.md

```diff
@@ -12,91 +12,229 @@ This section describes how to handle planned node maintenance or upgrading Kuber
 - [Upgrading Kubernetes](#upgrading-kubernetes)
   - [In-place Upgrade](#in-place-upgrade)
   - [Managed Kubernetes](#managed-kubernetes)
+- [Node Drain Policy Recommendations](#node-drain-policy-recommendations)
 
 ## Updating the Node OS or Container Runtime
 
 1. Cordon the node. Longhorn will automatically disable the node scheduling when a Kubernetes node is cordoned.
 
 1. Drain the node to move the workload to somewhere else.
 
-    You will need to use `--ignore-daemonsets` to drain the node.
-    The `--ignore-daemonsets` is needed because Longhorn deployed some daemonsets such as `Longhorn manager`, `Longhorn CSI plugin`, `engine image`.
+   You will need to use `--ignore-daemonsets` to drain the node. The `--ignore-daemonsets` is needed because Longhorn
```

Comment:
~You will need to use~ 
It will be necessary to use ...

Reply:
Fixing in an amend.